common/configurable/src/main/java/module-info.java

```diff
@@ -24,6 +24,7 @@
     requires io.helidon.builder.api;
     requires io.helidon.common.context;
     requires java.management;
+    requires java.logging;
```

Comment:
This should not be done. JUL is only used in tests.

Reply:
This is a problem of your configuration of IDE. It should work just fine when you run it from command line

Reply:
In Idea, you need to "modify options", and "do not use --module-path" option in your runner configuration, otherwise it uses the module of the main source, and will not work as expected.

Reply:
This error comes from command line, I always use it. Mockito is not able to mock final classes (I think the limitation comes from proxied classes).

Even if that test was enabled, still the test is not correct because I need to set `System.Logger log` in the class, and this is `static final`. I would need to modify the class to be able to test it somehow and I don't see a nice way to do it. Probably the nicest is to have a provider that returns the log instance, so I can inject this provider in the test.

Testing that a warning log is written is not really that important. I still think we can remove the test.